transmitterIO.js

```diff
@@ -324,12 +324,16 @@ module.exports = async (options, storage, storageLock, client) => {
       });
   };
 
-  const sendNewGlucose = (sgv, sendSGV) => {
+  const sendNewGlucose = async (sgv, sendSGV) => {
     client.newSGV(sgv);
 
     if (!sgv.glucose) {
       // Set to 5 so NS will plot the unfiltered glucose values
       sgv.glucose = 5;
+    } else {
+      // wait for fakeMeter to finish so it doesn't interfere with
+      // pump-loop
+      await fakeMeter.glucose(sgv.glucose);
```

Comment:
Perfect - this is the best possible spot to run fakemeter to avoid potential pump communication collisions. It will still collide on occasion, but the only ill effect of a sporadic collision is that fakemeter will miss posting a single offline glucose reading to the pump.